openpdf/src/main/java/com/lowagie/text/pdf/FontDetails.java

```diff
@@ -49,6 +49,7 @@
 
 package com.lowagie.text.pdf;
 
+import java.awt.font.GlyphVector;
```

Comment:
I'm wondering how long are the awt classes being support by the JDK. Just a non relevant comment.